client/src/pages/Messages.tsx

```diff
@@ -48,59 +271,23 @@ const Profile = () => {
 
   const handleFormSubmit: React.FormEventHandler<HTMLFormElement> = (e) => {
     e.preventDefault()
+    const form = e.target as any
 
-    const formData = new FormData(e.target as any)
-    const action = formData.get('action') as string
-    const payloadStr = formData.get('payload') as string
-    const metadataStr = formData.get('metadata') as string
-
-    if (!action) {
-      Log.warn(
-        { action, payloadStr, metadataStr },
-        'Missing required message arguments',
-      )
-      return
-    }
-
-    if (!payloadStr) {
-      Log.warn(
-        { action, payloadStr, metadataStr },
-        'Missing required message arguments',
-      )
-      return
-    }
-
-    if (!metadataStr) {
-      Log.warn(
-        { action, payloadStr, metadataStr },
-        'Missing required message arguments',
-      )
-
-      return
-    }
-
-    let payload, metadata
-
-    try {
-      payload = JSON.parse(payloadStr)
-      metadata = JSON.parse(payloadStr)
-    } catch (e) {
-      Log.warn(
-        { err: e, payloadStr, metadataStr },
-        'Error parsing payload or metadata ',
-      )
-
-      return
-    }
+    const formData = new FormData(form)
+    const message = formData.get('message') as string
 
     sendMessage({
-      action,
-      payload,
-      metadata,
+      action: '@@MESSAGES/SEND',
+      payload: {
+        receiverId: 'ee8886dc-d78b-4c0a-9650-3c19f6661de7',
+        message,
+      },
+      metadata: {
+        authorId: 'ee8886dc-d78b-4c0a-9650-3c19f6661de7',
```

Comment:
We will need to change this to be pulled from some state. This is the _internal_ ID, not the external ID